test/powershell/Modules/Microsoft.PowerShell.Management/Get-ComputerInfo.Tests.ps1

```diff
@@ -341,7 +341,9 @@ public static extern int LCIDToLocaleName(uint localeID, System.Text.StringBuild
         $returnValue = $null
         try
         {
-            $returnValue = Get-CimClassPropVal Win32_DeviceGuard $propertyName -namespace 'root\Microsoft\Windows\DeviceGuard' -ErrorAction Stop
+            #Since we want to catch any exceptions here, we won't use the above Get-Cim* helper functions
```

Comment:
This comment no longer makes sense with the removal of the previous line.  I don't see any value in having this comment, just remove it.